src/server/zset_family.cc

```diff
@@ -3042,24 +3104,32 @@ void ZSetFamily::GeoRadiusByMember(CmdArgList args, ConnectionContext* cntx) {
   string_view key = ArgS(args, 0);
   // member to latlong, set shape.xy
   string_view member = ArgS(args, 1);
+  OpResult<double> member_score;
+
   auto cb = [&](Transaction* t, EngineShard* shard) {
-    return OpScore(t->GetOpArgs(shard), key, member);
+    member_score = OpScore(t->GetOpArgs(shard), key, member);
+    return OpStatus::OK;
   };
-  OpResult<double> result = cntx->transaction->ScheduleSingleHopT(std::move(cb));
-  if (result.status() == OpStatus::WRONG_TYPE) {
+  cntx->transaction->Schedule();
+  cntx->transaction->Execute(std::move(cb), false);
+  if (member_score.status() == OpStatus::WRONG_TYPE) {
+    cntx->transaction->Conclude();
     return cntx->SendError(kWrongTypeErr);
-  } else if (!result) {
+  } else if (!member_score) {
+    cntx->transaction->Conclude();
```

Comment:
member_score is null and fail intermittently as mentioned here: https://github.com/dragonflydb/dragonfly/issues/1548#issuecomment-1869839316

dataset:
`geoadd Europe 13.4050 52.5200 Berlin 3.7038 40.4168 Madrid 9.1427 38.7369 Lisbon 2.3522 48.8566 Paris 16.3738 48.2082 Vienna 4.8952 52.3702 Amsterdam 10.7522 59.9139 Oslo 23.7275 37.9838 Athens 19.0402 47.4979 Budapest 6.2603 53.3498 Dublin`

Not sure if I have registered this command correctly.